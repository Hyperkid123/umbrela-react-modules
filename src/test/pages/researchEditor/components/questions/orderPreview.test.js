import React from 'react'
import OrderPreview from '../../../../../pages/researchEditorModule/components/questions/orderPreview';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';

describe('Order Preview component', () => {
    it('should render correctly empty', () => {
      const tree = shallow(
        <OrderPreview
          options={[]}
        />
      ).dive();
      expect(toJson(tree)).toMatchSnapshot();
    });

    it('should render correctly with options', () => {
      const tree = shallow(
        <OrderPreview
          options={[{
            title: 'foo',
            optionId: 1
          }, {
            title: 'foo2',
            optionId: 2
          }]}
        />
      ).dive();
      expect(toJson(tree)).toMatchSnapshot();
    });
});
