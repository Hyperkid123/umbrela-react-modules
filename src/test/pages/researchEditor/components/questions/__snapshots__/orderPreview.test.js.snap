// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Order Preview component should render correctly empty 1`] = `
<div
  className="sc-bdVaJa fizmbb"
>
  <styled.div>
    <styled.div />
  </styled.div>
</div>
`;

exports[`Order Preview component should render correctly with options 1`] = `
<div
  className="sc-bdVaJa fizmbb"
>
  <styled.div>
    <styled.div>
      <styled.div
        key="1"
      >
        <pure(DragHandle) />
        <span>
          foo
        </span>
      </styled.div>
      <styled.div
        key="2"
      >
        <pure(DragHandle) />
        <span>
          foo2
        </span>
      </styled.div>
    </styled.div>
  </styled.div>
</div>
`;
